OpenLIFUData/OpenLIFUData.py

```diff
@@ -216,6 +271,28 @@ def on_item_double_clicked(self, index : qt.QModelIndex):
         # Make sure parameter node is initialized (needed for module reload)
         self.initializeParameterNode()
 
+    @display_errors
+    def onAddNewSubjectClicked(self, checked:bool) -> None:
+
+        if not self.logic.db:
+            slicer.util.errorDisplay("Database needs to be loaded first")
```

Comment:
Suggest disabling the button when there is no databse rather than allowing it to be pressed and displaying this error.

Similar to `update_sessionLoadButton_enabled`, you can have a `update_newSubjectButton_enabled` which checks logic.db and updates enabledness and sets a tooltip

`update_newSubjectButton_enabled` could be called at the end of `onLoadDatabaseClicked`